### ISSUE TITLE
Peer panics in recommitLostBlocks() when processing a collection configuration change

### ISSUE THREAD

**Issue by denyeart** [hyperledger/fabric#4229]
### Description

If peer is in recommitLostBlocks() and needs to process a chaincode deployment with collection configuration change, it will panic with the following error.

```
2023-05-18 15:12:36.758 EDT 00ac INFO [kvledger] recommitLostBlocks -> Recommitting lost blocks - firstBlockNum=5, lastBlockNum=5, recoverables=[]kvledger.recoverable{(*txmgr.LockBasedTxMgr)(0xc000347180)}
2023-05-18 15:12:36.758 EDT 00ad DEBU [blkstorage] retrieveBlockByNumber -> retrieveBlockByNumber() - blockNum = [5]
2023-05-18 15:12:36.758 EDT 00ae DEBU [blkstorage] newBlockfileStream -> newBlockfileStream(): filePath=[/Users/enyeart/work/src/github.com/hyperledger/fabric-samples/test-network-nano-bash/data/peer0.org1.example.com/ledgersData/chains/chains/mychannel/blockfile_000000], startOffset=[69302]
2023-05-18 15:12:36.759 EDT 00af DEBU [blkstorage] nextBlockBytesAndPlacementInfo -> Remaining bytes=[5332], Going to peek [8] bytes
2023-05-18 15:12:36.759 EDT 00b0 DEBU [blkstorage] nextBlockBytesAndPlacementInfo -> Returning blockbytes - length=[5330], placementInfo={fileNum=[0], startOffset=[69302], bytesOffset=[69304]}
2023-05-18 15:12:36.759 EDT 00b1 DEBU [pvtdatastorage] GetPvtDataByBlockNum -> Get private data for block [5], filter=ledger.PvtNsCollFilter(nil)
2023-05-18 15:12:36.759 EDT 00b2 DEBU [pvtdatastorage] GetPvtDataByBlockNum -> Querying private data storage for write sets using startKey=[]byte{0x2, 0x1, 0x5, 0x0}, endKey=[]byte{0x2, 0x1, 0x6, 0x0}
2023-05-18 15:12:36.759 EDT 00b3 DEBU [leveldbhelper] GetIterator -> Getting iterator for range [[]byte{0x6d, 0x79, 0x63, 0x68, 0x61, 0x6e, 0x6e, 0x65, 0x6c, 0x0, 0x2, 0x1, 0x5, 0x0}] - [[]byte{0x6d, 0x79, 0x63, 0x68, 0x61, 0x6e, 0x6e, 0x65, 0x6c, 0x0, 0x2, 0x1, 0x6, 0x0}]
2023-05-18 15:12:36.759 EDT 00b4 DEBU [lockbasedtxmgr] CommitLostBlock -> Constructing updateSet for the block 5
2023-05-18 15:12:36.759 EDT 00b5 DEBU [lockbasedtxmgr] ValidateAndPrepare -> Waiting for purge mgr to finish the background job of computing expirying keys for the block
2023-05-18 15:12:36.759 EDT 00b6 DEBU [lockbasedtxmgr] ValidateAndPrepare -> lock acquired on oldBlockCommit for validating read set version against the committed version
2023-05-18 15:12:36.759 EDT 00b7 DEBU [lockbasedtxmgr] ValidateAndPrepare -> Validating new block with num trans = [1]
2023-05-18 15:12:36.759 EDT 00b8 DEBU [validation] ValidateAndPrepareBatch -> ValidateAndPrepareBatch() for block number = [5]
2023-05-18 15:12:36.759 EDT 00b9 DEBU [validation] ValidateAndPrepareBatch -> preprocessing ProtoBlock...
2023-05-18 15:12:36.759 EDT 00ba DEBU [validation] preprocessProtoBlock -> txType=ENDORSER_TRANSACTION
2023-05-18 15:12:36.759 EDT 00bb DEBU [validation] validateAndPrepareBatch -> Block [5] Transaction index [0] TxId [ec2c9df453cb528c3f4b210331e04a569f1cf7f3428d62a207c4290ddd73885c] marked as valid by state validator. ContainsPostOrderWrites [false]
2023-05-18 15:12:36.759 EDT 00bc DEBU [validation] applyWriteSet -> txops=validation.txOps{validation.compositeKey{ns:"_lifecycle", coll:"", key:"namespaces/fields/private/Collections"}:(*validation.keyOps)(0xc00067a0c0), validation.compositeKey{ns:"_lifecycle", coll:"", key:"namespaces/fields/private/Sequence"}:(*validation.keyOps)(0xc00067a100)}
2023-05-18 15:12:36.759 EDT 00bd DEBU [validation] ValidateAndPrepareBatch -> validating rwset...
2023-05-18 15:12:36.759 EDT 00be DEBU [validation] ValidateAndPrepareBatch -> postprocessing ProtoBlock...
2023-05-18 15:12:36.759 EDT 00bf DEBU [validation] ValidateAndPrepareBatch -> ValidateAndPrepareBatch() complete
2023-05-18 15:12:36.759 EDT 00c0 INFO [lifecycle] update -> Updating cached definition for chaincode 'private' on channel 'mychannel'
2023-05-18 15:12:36.759 EDT 00c1 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/metadata/private
2023-05-18 15:12:36.759 EDT 00c2 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/EndorsementInfo
2023-05-18 15:12:36.759 EDT 00c3 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/ValidationInfo
2023-05-18 15:12:36.760 EDT 00c4 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key=>"Ho�}3
b�]�.��9U�,UӜ���
k
Cf&��05-18 15:12:36.760 EDT 00c5 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key=�O1�g��:��edf��l�V���	V
2023-05-18 15:12:36.760 EDT 00c6 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key=�\Xĝ��q�Q\z�O�PA�8T�����}
2023-05-18 15:12:36.760 EDT 00c7 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key=�ڹ���3������mDO��.�'+�V7
è023-05-18 15:12:36.760 EDT 00c8 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key= ��к��Lv0G'���,n9ĨH�~g
�Z�fW�JA@j�����5���*Y���EDT 00c9 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle$$h_implicit_org_Org1MSP, key=A��Ũ
2023-05-18 15:12:36.760 EDT 00ca INFO [lifecycle] update -> Chaincode with package ID 'private:98669c58ff1e2edfccd92ed5fd62e88ef7c3bf0fd571bba9ebb06ddd74e8960f' now available on channel mychannel for chaincode definition private:1.0
2023-05-18 15:12:36.760 EDT 00cb DEBU [lifecycle] ProcessApproveOrDefineEvent -> processApproveOrDefineEvent() channelID=mychannel chaincodeName=private cachedChaincode="{sequence: 2, endorsement info: (version: '1.0', plugin: 'escc', init required: false), validation info: (plugin: 'vscc', policy: '12202f4368616e6e656c2f4170706c69636174696f6e2f456e646f7273656d656e74'), collections: (config:<static_collection_config:<name:\"assetCollection\" member_orgs_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > rules:<signed_by:1 > > > identities:<principal:\"\\n\\007Org1MSP\" > identities:<principal:\"\\n\\007Org2MSP\" > > > required_peer_count:1 maximum_peer_count:1 block_to_live:1000000 member_only_read:true member_only_write:true endorsement_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > rules:<signed_by:1 > > > identities:<principal:\"\\n\\007Org1MSP\" > identities:<principal:\"\\n\\007Org2MSP\" > > > > > config:<static_collection_config:<name:\"Org1MSPPrivateCollection\" member_orgs_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > > > identities:<principal:\"\\n\\007Org1MSP\" > > > maximum_peer_count:1 block_to_live:3 member_only_read:true endorsement_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > > > identities:<principal:\"\\n\\007Org1MSP\" > > > > > config:<static_collection_config:<name:\"Org2MSPPrivateCollection\" member_orgs_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > > > identities:<principal:\"\\n\\007Org2MSP\" > > > maximum_peer_count:1 block_to_live:3 member_only_read:true endorsement_policy:<signature_policy:<rule:<n_out_of:<n:1 rules:<signed_by:0 > rules:<signed_by:1 > > > identities:<principal:\"\\n\\007Org1MSP\" > identities:<principal:\"\\n\\007Org2MSP\" > > > > > ) true 0xc000766140 [\ufffdO1\u0006\ufffdg\ufffd\ufffd:\ufffd\ufffdedf\ufffd\u0002\ufffdl\ufffdV\ufffd\u0017\ufffd\ufffd\tV\rCf&\ufffd\ufffd \ufffd\\X\u000eĝ\ufffd\u0010\ufffdq\ufffdQ\\z\ufffdO\ufffd\ufffd\u0008PA\ufffd8\u001aT\ufffd\u0003\ufffd\ufffd\ufffd\ufffd} \ufffdڹ\ufffd\ufffd\u000f\ufffd\ufffd\ufffd3\ufffd\ufffd\ufffd\ufffd\ufffd\ufffd\ufffd\u0013mDO\ufffd\ufffd.\u0010\u0014\ufffd'+\ufffdV7  \ufffd\ufffdк\u0016\ufffd\ufffdLv0G'\ufffd\ufffd\u0007\ufffd\ufffd,\u0003n9ĨH\ufffd~g\rè\u0019 >\"Ho\ufffd}3\nb\ufffd]\ufffd.\ufffd\u001f\ufffd9U\ufffd\ufffd,UӜ\ufffd\u0013\ufffd\u0010\u000f\ufffd\nk]}"
2023-05-18 15:12:36.799 EDT 00cc DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/
2023-05-18 15:12:36.800 EDT 00cd DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/couchdb/
2023-05-18 15:12:36.800 EDT 00ce DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/couchdb/collections/
2023-05-18 15:12:36.800 EDT 00cf DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/couchdb/collections/assetCollection/
2023-05-18 15:12:36.800 EDT 00d0 DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/couchdb/collections/assetCollection/indexes/
2023-05-18 15:12:36.800 EDT 00d1 DEBU [ccprovider] MetadataAsTarEntries -> Wrote file to statedb tar: META-INF/statedb/couchdb/collections/assetCollection/indexes/indexOwner.json
2023-05-18 15:12:36.900 EDT 00d2 DEBU [ccprovider] MetadataAsTarEntries -> Created metadata tar
2023-05-18 15:12:36.900 EDT 00d3 WARN [lifecycle] handleMetadataUpdatesForChannel -> no metadata found on channel 'mychannel': could not get channel config for channel 'mychannel'
2023-05-18 15:12:36.900 EDT 00d4 DEBU [lockbasedtxmgr] invokeNamespaceListeners -> Invoking listener for state changes:lifecycle cache listener
2023-05-18 15:12:36.901 EDT 00d5 DEBU [cceventmgmt] HandleStateUpdates -> Channel [mychannel]: Handling state updates in LSCC namespace - stateUpdates=map[string][]*kvrwset.KVWrite{"_lifecycle":[]*kvrwset.KVWrite{(*kvrwset.KVWrite)(0xc0004e80f0), (*kvrwset.KVWrite)(0xc0004e8140)}}
2023-05-18 15:12:36.901 EDT 00d6 INFO [cceventmgmt] HandleStateUpdates -> Channel [mychannel]: Handling deploy or update of chaincode [private]
2023-05-18 15:12:36.901 EDT 00d7 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/metadata/private
2023-05-18 15:12:36.901 EDT 00d8 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/EndorsementInfo
2023-05-18 15:12:36.901 EDT 00d9 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/ValidationInfo
2023-05-18 15:12:36.901 EDT 00da DEBU [cceventmgmt] HandleChaincodeDeploy -> Channel [mychannel]: Handling chaincode deploy event for chaincode [[]]
2023-05-18 15:12:36.901 EDT 00db DEBU [lockbasedtxmgr] invokeNamespaceListeners -> Invoking listener for state changes:lscc state listener
=== HandleStateUpdates ===
2023-05-18 15:12:36.901 EDT 00dc DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/metadata/private
2023-05-18 15:12:36.901 EDT 00dd DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/Sequence
2023-05-18 15:12:36.901 EDT 00de DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/EndorsementInfo
2023-05-18 15:12:36.901 EDT 00df DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/ValidationInfo
2023-05-18 15:12:36.901 EDT 00e0 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/Collections
2023-05-18 15:12:36.901 EDT 00e1 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/metadata/private
2023-05-18 15:12:36.901 EDT 00e2 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/EndorsementInfo
2023-05-18 15:12:36.901 EDT 00e3 DEBU [stateleveldb] GetState -> GetState(). ns=_lifecycle, key=namespaces/fields/private/ValidationInfo
=== Calling to elgEnabledCollNames() ===
=== accessPolicy:
(*cauthdsl.policy)(0xc0004f0a60)({
 signaturePolicyEnvelope: (*common.SignaturePolicyEnvelope)(0xc0004e88c0)(rule:<n_out_of:<n:1 rules:<signed_by:0 > > > identities:<principal:"\n\007Org2MSP" > ),
 evaluator: (func([]msp.Identity, []bool) bool) 0x100686b20,
 deserializer: (msp.IdentityDeserializer) <nil>
})
2023-05-18 15:12:36.901 EDT 00e4 INFO [gossip.gossip] Stop -> Stopping gossip
2023-05-18 15:12:36.901 EDT 00e5 DEBU [gossip.gossip] handlePresumedDead -> Exiting
2023-05-18 15:12:36.901 EDT 00e6 DEBU [gossip.gossip] acceptMessages -> Exiting
2023-05-18 15:12:36.901 EDT 00e7 DEBU [gossip.discovery] handleMessages -> Stopped
2023-05-18 15:12:36.901 EDT 00e8 INFO [gossip.discovery] Stop -> Stopping
2023-05-18 15:12:36.901 EDT 00ea INFO [gossip.discovery] Stop -> Stopped
2023-05-18 15:12:36.901 EDT 00e9 DEBU [gossip.discovery] handleEvents -> Stopped
2023-05-18 15:12:36.902 EDT 00eb INFO [gossip.comm] Stop -> Stopping
2023-05-18 15:12:36.902 EDT 00ec DEBU [gossip.comm] Stop -> Shut down connection store, connection count: 0
2023-05-18 15:12:36.902 EDT 00ed DEBU [gossip.comm] func1 -> Exiting Accept() loop
2023-05-18 15:12:36.902 EDT 00ee INFO [gossip.comm] Stop -> Stopped
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x10067e7c6]

goroutine 1 [running]:
github.com/hyperledger/fabric/common/policies.SignatureSetToValidIdentities({0xc0001a8a70, 0x1, 0x10000fe07?}, {0x0, 0x0})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/common/policies/policy.go:369 +0x126
github.com/hyperledger/fabric/common/cauthdsl.(*policy).EvaluateSignedData(0xc0004f0a60, {0xc0001a8a70?, 0xc0001a8008?, 0xc000723e08?})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/common/cauthdsl/policy.go:91 +0xa5
github.com/hyperledger/fabric/core/common/privdata.(*MembershipProvider).AmMemberOf(0xc00068efc0, {0xc000558540?, 0xc000723fa0?}, 0xc000344ae0)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/common/privdata/membershipinfo.go:65 +0x2d7
github.com/hyperledger/fabric/core/ledger/kvledger.(*collElgNotifier).elgEnabled(0xc000691a40, {0xc000558540, 0x9}, 0x18?, 0x0?)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/coll_elg_notifier.go:136 +0x39
github.com/hyperledger/fabric/core/ledger/kvledger.(*collElgNotifier).elgEnabledCollNames(0x1011a1300?, {0xc000558540, 0x9}, 0x1?, 0xc000766e80?)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/coll_elg_notifier.go:121 +0x258
github.com/hyperledger/fabric/core/ledger/kvledger.(*collElgNotifier).HandleStateUpdates(0xc000691a40, 0xc000766780)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/coll_elg_notifier.go:74 +0x390
github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr.(*LockBasedTxMgr).invokeNamespaceListeners(0xc000347180)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr/lockbased_txmgr.go:470 +0x3e7
github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr.(*LockBasedTxMgr).ValidateAndPrepare(0xc000347180, 0xc00032b080, 0x0?)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr/lockbased_txmgr.go:183 +0x38d
github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr.(*LockBasedTxMgr).CommitLostBlock(0xc002b320b0?, 0xc00032b080)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/txmgr/lockbased_txmgr.go:584 +0xc6
github.com/hyperledger/fabric/core/ledger/kvledger.(*kvLedger).recommitLostBlocks(0x100d216e0?, 0x5, 0x5, {0xc0003cffc0?, 0x1, 0x1})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger.go:481 +0x2db
github.com/hyperledger/fabric/core/ledger/kvledger.(*kvLedger).syncStateAndHistoryDBWithBlockstore(0xc002b320b0)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger.go:409 +0x3c5
github.com/hyperledger/fabric/core/ledger/kvledger.(*kvLedger).recoverDBs(0xc002b32138?)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger.go:354 +0x5c
github.com/hyperledger/fabric/core/ledger/kvledger.newKVLedger(0xc002b32000)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger.go:164 +0x5de
github.com/hyperledger/fabric/core/ledger/kvledger.(*Provider).open(0xc00068f110, {0xc000558540, 0x9}, 0x0, 0x0)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger_provider.go:378 +0x4a6
github.com/hyperledger/fabric/core/ledger/kvledger.(*Provider).Open(0xc00068f110, {0xc000558540, 0x9})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/kvledger/kv_ledger_provider.go:330 +0x127
github.com/hyperledger/fabric/core/ledger/ledgermgmt.(*LedgerMgr).OpenLedger(0xc0001ae540, {0xc000558540, 0x9})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/ledger/ledgermgmt/ledger_mgmt.go:211 +0x193
github.com/hyperledger/fabric/core/peer.(*Peer).Initialize(0xc0000001e0, 0xc000554920, 0x100fc9c2b?, {0x1011a1e80?, 0xc000115470}, {0x1011b1228, 0xc000690150}, {0x10119ff40, 0xc000350460}, {0x1011a4720, ...}, ...)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/core/peer/peer.go:512 +0x269
github.com/hyperledger/fabric/internal/peer/node.serve({0x100a5ebc0?, 0xc000021d18?, 0xc000021cf0?})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/internal/peer/node/start.go:770 +0x4e9a
github.com/hyperledger/fabric/internal/peer/node.glob..func6(0x101a46f40?, {0x101a87360?, 0x0?, 0x0?})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/internal/peer/node/start.go:129 +0x56
github.com/spf13/cobra.(*Command).execute(0x101a46f40, {0x101a87360, 0x0, 0x0})
	/Users/enyeart/work/src/github.com/hyperledger/fabric/vendor/github.com/spf13/cobra/command.go:762 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x101a478c0)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/vendor/github.com/spf13/cobra/command.go:852 +0x2dc
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/enyeart/work/src/github.com/hyperledger/fabric/vendor/github.com/spf13/cobra/command.go:800
main.main()
	/Users/enyeart/work/src/github.com/hyperledger/fabric/cmd/peer/main.go:56 +0x392
```

### Steps to reproduce

- Deploy two versions of a chaincode. In the second version, update the collection configuration membership.
- Stop peer.
- Delete state database so that state must get recommitted upon restart.
- Restart peer.
- Observe the panic above.

The problem happens because the identityDeserializer is null at line:
https://github.com/hyperledger/fabric/blob/v2.4.5/common/policies/policy.go#L368

The debug above shows that accessPolicy.deserializer is null at line:
https://github.com/hyperledger/fabric/blob/v2.4.5/core/common/privdata/membershipinfo.go#L54

In the recommitLostBlocks() scenario, the deserializer must not be initialized yet.

The problem appeared on v2.4.5 but I expect the problem still exists in main branch.

**Comment by yacovm** [comment]
It's because the MSP manager is initialized at [createChannel](https://github.com/hyperledger/fabric/blob/main/core/peer/peer.go#L519) but you [initialize the ledger](https://github.com/hyperledger/fabric/blob/main/core/peer/peer.go#LL512C30-L512C40) before that.

**Comment by denyeart** [comment]
Fix backported to v2.2 and v2.5.
